CONTRIBUTOR_LADDER.md

```diff
@@ -60,26 +68,32 @@ A Maintainer must meet the rights, responsiblities, and requirements of a Contri
     * Determining strategy and policy for the project
     * Participating in, and leading, community meetings
     * Helps run the project infrastructure
-* Requirements
+* Requirements:
     * Experience as a Contributor for at least 3 months
-    * Demonstrates a broad knowledge of the project across multiple areas
+    * Deep understanding of the technical goals and direction of the project
+    * Deep understanding of the technical domain of the project across multiple areas
+    * Sustained contributions to design and direction by doing all of:
+      * Authoring and reviewing proposals (GitHub Issues for refactoring, enhancements, or new functionality)
+      * Initiating, contributing, and resolving discussions (emails, GitHub issues, meetings)
+      * Identifying subtle or complex issues in designs and implementation PRs
     * Is able to exercise judgement for the good of the project, independent of their employer, social circles, or teams
     * Mentors other Contributors
 * Additional privileges:
+    * Approve PRs to any area of the project
     * Represent the project in public as a Maintainer
     * Communicate with the CNCF on behalf of the project
     * Have a vote in Maintainer decisions
 
-Process of becoming a maintainer:
+Process of becoming a Maintainer:
 
-1. Any current Maintainer may nominate a current Contributor to become a new Maintainer, by opening a PR against the root of the [OSM repository](https://github.com/openservicemesh/osm) and adding the nominee to the [OWNERS file](https://github.com/openservicemesh/osm/blob/main/OWNERS). Additionally, if the nomination is for a code owner whose PR approvals are meant to satisfy the PR merge requirements, the nominee must be added to the [CODEOWNERS file](https://github.com/openservicemesh/osm/blob/main/CODEOWNERS) in the same PR.
+1. A current contributor may be self-nominated or be nominated by a current Maintainer by opening a PR against the root of the [OSM repository] and adding the nominee to the [OWNERS file] under the *owners* entry. Additionally, if the nomination is for a code owner whose PR approvals are meant to satisfy the PR merge requirements, the nominee must be added to the [CODEOWNERS file] in the same PR. A codeowner is defined per the [GitHub definition](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#about-code-owners).
 2. The nominee will add a comment to the PR testifying that they agree to all requirements of becoming a Maintainer.
 3. A majority of the current Maintainers must then approve the PR.
 
```

Comment:
We could also define the process of how to add areas of ownership.

Reply:
Added an example here: https://github.com/openservicemesh/osm/pull/4776/files#diff-653bc99dc2e48c5310ac61677b097abbaa6193149342b09bc721e058e3aae91bR91